resources/options_lookup.tsv

```diff
@@ -869,6 +869,10 @@ Heating Setpoint	73F	ResidentialHVACHeatingSetpoints	weekday_setpoint=73	weekend
 Heating Setpoint	73F w/setback	ResidentialHVACHeatingSetpoints	weekday_setpoint=73	weekend_setpoint=73	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
 Heating Setpoint	75F	ResidentialHVACHeatingSetpoints	weekday_setpoint=75	weekend_setpoint=75	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
 Heating Setpoint	75F w/setback	ResidentialHVACHeatingSetpoints	"weekday_setpoint=68,68,68,68,68,68,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,68"	"weekend_setpoint=68,68,68,68,68,68,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,75,68"	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	62F	ResidentialHVACHeatingSetpoints	weekday_setpoint=62	weekend_setpoint=62	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	64F	ResidentialHVACHeatingSetpoints	weekday_setpoint=64	weekend_setpoint=64	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	66F	ResidentialHVACHeatingSetpoints	weekday_setpoint=66	weekend_setpoint=66	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	67F	ResidentialHVACHeatingSetpoints	weekday_setpoint=67	weekend_setpoint=67	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
 Cooling Setpoint	FIXME Location Average	ResidentialHVACCoolingSetpoints	weekday_setpoint=76	weekend_setpoint=76	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							

```

Comment:
Can we now remove "FIXME Location Average" options from the Heating Setpoint and Cooling Setpoint parameters?

Reply:
@shorowit, I believe we could? @ejhw?

Reply:
If no projects are using it anymore, then it can be removed.